examples/preprocessing/example_date_featurizer.py

```diff
@@ -45,14 +45,14 @@
 
 # Here's what the featurizer will create for us:
 date_feat = preprocessing.DateFeaturizer(
-    column_name="date",  # the name of the date feature in the exog matrix
+    column_name="date",  # the name of the date feature in the X matrix
     with_day_of_week=True,
     with_day_of_month=True)
 
 _, X_train_feats = date_feat.fit_transform(y_train, X_train)
-print("Head of generated exog features:\n%s" % repr(X_train_feats.head()))
+print("Head of generated X features:\n%s" % repr(X_train_feats.head()))
```

Comment:
This could be an f-string (a .format() might be better, considering what has to be done to get it to a string), but not a hill I'm willing to die on

Reply:
True, not sure I want to mess with the examples yet since our doc base builds that and I can't recall if it's python 3.6 or 3.5

Reply:
3.7 🤪


https://github.com/alkaline-ml/pmdarima-docker/blob/master/doc-base/Dockerfile